packaging/hudi-utilities-bundle/pom.xml

```diff
@@ -376,6 +376,12 @@
       <groupId>org.apache.parquet</groupId>
       <artifactId>parquet-avro</artifactId>
       <scope>compile</scope>
+      <exclusions>
+        <exclusion>
+          <groupId>org.xerial.snappy</groupId>
+          <artifactId>snappy-java</artifactId>
```

Comment:
does this break the snappy compression functionality in `hudi-utilities-bundle`?

Reply:
No idea. Is there a good way to test? Any additional info you know about this would be helpful to me. Thanks

Reply:
We tested in TestHoodieSparkSqlWriter.scala and also added a param to getCommonParams that sets "hoodie.parquet.compression.codec" to snappy and ran all the tests and they all passed

Reply:
we test this and things are good.